docs/guide/install-and-run.asciidoc

```diff
@@ -142,7 +142,7 @@ and configuring it with the address of your APM Server, a secret token (if neces
 
 |=======================================================================
 
-TIP: Check the <<agent-server-compatibility,Agent/Server compatibility matrix>> to ensure you're using agents that are compatible with your version of Elasticsearch.
+TIP: Check the {apm-py-ref-v}/agent-server-compatibility.html[Agent/Server compatibility matrix] to ensure you're using agents that are compatible with your version of Elasticsearch.
```

Comment:
```suggestion
TIP: Check the {apm-overview-ref-v}/agent-server-compatibility.html[Agent/Server compatibility matrix] to ensure you're using agents that are compatible with your version of Elasticsearch.
```

Reply:
Wrong book

Reply:
Once you make this update and the tests pass, feel free to merge this and open the backports. If for some reason you can't merge, most of the Server devs are in your time zone. Just post in #apm-server

Reply:
Thanks